tests/jballerina-unit-test/src/test/resources/test-src/error/error_test.bal

```diff
@@ -412,10 +412,10 @@ function bar2(){
     bar();
 }
 
-function testStackOverFlow() returns [error:CallStackElement[], string]? {
+function testStackOverFlow() returns [error:StackFrame[], string]? {
     error? e = trap bar();
     if (e is error){
-        return [e.stackTrace().callStack, e.message()];
```

Comment:
This is also a backward incompatible change, right?

Reply:
I mean not being able to access `.callStack`?

Reply:
yes. we no longer returns CallStack class